plugins/MantisGraph/MantisGraph.php

```diff
@@ -140,15 +149,28 @@ function graph_filter_menu() {
 	function resources() {
 		if( current( explode( '/', gpc_get_string( 'page', '' ) ) ) === $this->basename ) {
 			if( config_get_global( 'cdn_enabled' ) == ON ) {
```

Comment:
Initialize the `$t_scripts` array in this branch.

Reply:
Not necessary. `$a[] = 1;` will properly initialize the array :
``` 
php > var_dump($a);
PHP Notice:  Undefined variable: a in php shell code on line 1
NULL
php > $a[]=1;
php > var_dump($a);
array(1) {
  [0] =>
  int(1)
}
```